sourcegraph/sourcegraph
# [SG-35698] Fix HeroPage icon link missing a label

### Audit type
ARC Toolkit

### User journey audit issue [#34112](https://github.com/sourcegraph/sourcegraph/issues/34112)

## Description
When using the HeroPage react component, links on the top of the page are represented as an image. If we use the sourcegraph logo icon there, we have 2 problems as a result:

- The svg is focusable
- The link does not contain any text or aria-label, so the screen reader cannot interpret where the link goes to or what is the purpose of the link

## Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/35698)
[GitStart Issue](https://app.gitstart.com/clients/sourcegraph/tickets/SG-35698)

## Implementation Details
- [x] The svg itself is not focusable
- [x] The link contains a label or description, that is not visible by default. If the description/label was not provided, the label should contain the url the link points to

## Test plan
To test this PR, 
- Run app on branch and open [the issue link](https://github.com/sourcegraph/sourcegraph/issues/35698)
- Confirm that hero page icons now has a label or hidden
- Confirm there are no UI diffs

## App preview:

- [Web](https://sg-web-contractors-sg-35698.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-ohyreyhnjk.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
